L60-core-remove-custom-allocator.md

```diff
@@ -0,0 +1,91 @@
+Remove grpc_use_signal from core surface API
```

Comment:
Looks like the title needs to be updated.

Reply:
Oops. Done!